src/components/Markdown/index.tsx

```diff
@@ -15,9 +19,9 @@ interface IMarkdownProps {
   mainly to provide styles to the html generated.
 */
 export const Markdown: FC<IMarkdownProps> = ({
-  content,
-  ...restProps
-}) => {
+                                               content,
+                                               ...restProps
+                                             }) => {
```

Comment:
You can add Prettier on save in your editor, or run `npm run format` :)

Reply:
Yeah, no worries, I accidently copy pasted my branch which wasn't rebased so it pushed those changes. Undid it. should be good now